src/main/java/svc/data/citations/datasources/tyler/TylerTransformer.java

```diff
@@ -1,5 +1,154 @@
 package svc.data.citations.datasources.tyler;
 
-public class TylerTransformer {
+import java.math.BigDecimal;
+import java.sql.ResultSet;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
 
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Component;
+
+import svc.data.jdbc.BaseJdbcDao;
+import svc.logging.LogSystem;
+import svc.models.Citation;
+import svc.models.Court;
+import svc.models.Violation;
+import svc.types.HashableEntity;
+
+@Component
+public class TylerTransformer extends BaseJdbcDao {
+
+	private DateTimeFormatter localDateFormatter = DateTimeFormatter.ofPattern("MM/dd/uuuu");
+
+	public List<Citation> citationsFromTylerCitations(List<TylerCitation> tylerCitations) {
+		if (tylerCitations != null) {
+			return tylerCitations.stream()
+					.map(tylerCitation -> citationFromTylerCitation(tylerCitation))
+					.collect(Collectors.toList());
+		}
+		return null;
+	}
+
+	private LocalDateTime parseViolationCourtDate(String violationCourtDateString) {
+		try {
+			return LocalDateTime.parse(violationCourtDateString);
+		} catch (DateTimeParseException ex) {
+		}
```

Comment:
Yeah. I left it empty because there's nothing to do and we don't have a solid logging strategy in place. We can bring in the log system, and log there. It looked like we were moving away from that (i saw it removed from other files). Eventually we should actually set up logback or log4j correctly and just use normal java logging.